updateutils/info.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "updateutils",
+    "short": "Update Jojo's utils",
+    "description": "Update and check the version of Jojo's Utils.\nIf you want to see the command used, please follow this link: https://github.com/Just-Jojo/JojoCogs/blob/master/updateutils/update.py#L39",
+    "end_user_data_statement": "This cog does not store end user data.",
+    "install_msg": "Thank you for installing Jojo's update utils cog. This cog is designed for updating Jojo's utils easily.\n\nIf you want to see the command used, follow this link: https://github.com/Just-Jojo/JojoCogs/blob/master/updateutils/update.py#L39",
+    "author": [
+        "Jojo#7791"
+    ],
+    "required_cogs": {},
+    "requirements": [
+        "git+https://github.com/Just-Jojo/jojoutils.git"
+    ],
+    "tags": ["utils", "dev"],
+    "min_bot_version": "3.4.6",
+    "hidden": false,
```

```suggestion
    "hidden": true,
```
If you're going for approved, this kind of cog *should* remain hidden from the index